src/masternodes/mn_rpc.cpp

```diff
@@ -3086,39 +3089,326 @@ UniValue isappliedcustomtx(const JSONRPCRequest& request) {
+                        {
+                            {"address", RPCArg::Type::STR, RPCArg::Optional::NO, "The defi address is the key, the value is amount in amount@token format. "
+                                                                                     "If multiple tokens are to be transferred, specify an array [\"amount1@t1\", \"amount2@t2\"]"},
+                        },
+                    },
+                    {"selectionMode", RPCArg::Type::STR, /* default */ "pie", "If param \"from\" is empty this param indicates accaouts autoselection mode."
+                                                                              "May be once of:\n"
+                                                                              "  \"forward\" - Selecting accounts without sorting, just as address list sorted.\n"
+                                                                              "  \"crumbs\" - Selecting accounts by ascending of sum token amounts.\n"
+                                                                              "    It means that we will select first accounts with minimal sum of neccessary token amounts.\n"
+                                                                              "  \"pie\" - Selecting accounts by descending of sum token amounts.\n"
+                                                                              "    It means that we will select first accounts with maximal sum of neccessary token amounts."
+                    },
+                },
+                RPCResult{
+                       "\"hash\"                  (string) The hex-encoded hash of broadcasted transaction\n"
+                },
+                RPCExamples{
+                        HelpExampleCli("sendtokenstoaddress", "\"{}\" "
+                                                    "\"{\\\"dstAddress1\\\":\\\"1.0@DFI\\\",\\\"dstAddress2\\\":[\\\"2.0@BTC\\\", \\\"3.0@ETH\\\"]}\" "
+                                                    "\"crumbs\"") +
+                        HelpExampleCli("sendtokenstoaddress", "\"{\\\"srcAddress1\\\":\\\"2.0@DFI\\\",\\\"srcAddress2\\\":[\\\"3.0@DFI\\\", \\\"2.0@ETH\\\"]}\" "
+                                                    "\"{\\\"dstAddress1\\\":[\\\"5.0@DFI\\\", \\\"2.0@ETH\\\"]}\" \"\"")
+               },
+    }.Check(request);
+
+    if (pwallet->chain().isInitialBlockDownload()) {
+        throw JSONRPCError(RPC_CLIENT_IN_INITIAL_DOWNLOAD, "Cannot create transactions while still in Initial Block Download");
+    }
+    pwallet->BlockUntilSyncedToCurrentChain();
+
+    RPCTypeCheck(request.params, {UniValue::VOBJ, UniValue::VOBJ, UniValue::VSTR}, false);
+
+    CAnyAccountsToAccountsMessage msg;
+
+    msg.to = DecodeRecipients(pwallet->chain(), request.params[1].get_obj());
+    const CBalances sumTransfersTo = SumAllTransfers(msg.to);
+    if (sumTransfersTo.balances.empty()) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "zero amounts");
```

Comment:
The error message can with more details, it is too short for user to find out what is wrong.

Reply:
Done